content/about/governance.pt.md

```diff
@@ -0,0 +1,32 @@
+---
+title: governance
+displayTitle: Governação
+description: 'Governação do Projeto | Node.js'
+authors: XhmikosR, fhemberger, thefourtheye, ryanmurakami, refack, nazarepiedady
+category: about
+---
+
+## Processo de Procura de Consenso
+
+O projeto da Node.js segue um modelo de tomada de decisão conhecido como [Procura de Consenso][consensus seeking].
+
+## Colaboradores
+
+O repositório principal [nodejs/node][] da GitHub é mantido pelos Colaboradores que são adicionados pelo Comissão Diretiva Técnica ([TSC, sigla em Inglês][tsc]) sobre uma base em curso.
+
+Os indivíduos fazendo contribuições significantes e valiosas são feitos Colaboradores e dado acesso de envio ao projeto. Estes indivíduos são identificados pela TSC e suas nomeações são discutidas com os Colaboradores existente.
```

Comment:
```suggestion
Os indivíduos fazendo contribuições significantes e valiosas são feitos Colaboradores e dado acesso de envio ao projeto. Estes indivíduos são identificados pela TSC e suas nomeações são discutidas com os Colaboradores existentes.
```

Reply:
@TiagoDanin thanks for noticing, I usually translate or write the
translations quickly so some little things can go out of the flow.

But do not worry, you can merge these additions because in the
next pull requests the suggestions you did will be added.

Reply:
@TiagoDanin the change suggested is already in the next
pull request so you do not need to worry about it. :blush:

Reply:
The ideal and we send it to `main` without any problems

You can add the new commits in `translate-to-portuguese` branch

Reply:
@TiagoDanin I understand!

I am uploading the updates.

Reply:
@TiagoDanin I already pushed the corrections. :+1: